code/modules/research/nanites/nanite_programs/buffing.dm

```diff
@@ -137,3 +137,36 @@
 	to_chat(host_mob, "<span class='notice'>Your body feels lighter and your legs feel relaxed!</span>")
 	host_mob.set_resting(FALSE)
 	host_mob.reagents.add_reagent(/datum/reagent/medicine/amphetamine, 3)
+
+/datum/nanite_program/armblade
+	name = "Nanite Blade"
+	desc = "The nanites form a sharp blade around the user's arm when activated."
+	use_rate = 1
+	rogue_types = list(/datum/nanite_program/necrotic, /datum/nanite_program/skin_decay)
+	var/obj/item/melee/arm_blade/nanite/blade
+
+/datum/nanite_program/armblade/enable_passive_effect()
```

Comment:
This needs an activation cooldown to prevent what will become an exploit where nanites spam it on and off to trigger damage and block the user's arm

Reply:
Done.